src/internal/modules/account/helpers.js

```diff
@@ -0,0 +1,33 @@
+const { throwIfError } = require('@envage/hapi-pg-rest-api');
+const internalConfig = require('internal/config');
+const services = require('internal/lib/connectors/services');
+
+const getEmailRegex = (config = internalConfig) => {
+  return (config.isLocal || config.testMode)
+    ? /(\.gov\.uk|gmail\.com)$/
+    : /\.gov\.uk$/;
+};
+
+const getUserById = async (userId) => {
```

Comment:
This looks like a useful function, maybe we should add this to the connector at `services.idm.users.findOneById` so it can be used elsewhere